Backend/app/provider-platform/driver-tracking-health-check/src/Service/Runner.hs

```diff
@@ -116,3 +111,11 @@ withLock serviceName func =
   Redis.withLockRedis key 10 (func `catch` (logError . makeLogSomeException))
   where
     key = "beckn:" <> serviceName <> ":lock"
+
+getAllDrivers :: (Redis.HedisFlow m r, MonadReader r m) => Seconds -> UTCTime -> m [(Text, Maybe FCM.FCMRecipientToken)]
+getAllDrivers locationDelay now = do
+  allDriversLastLocation <- Redis.withCrossAppRedis $ Redis.hGetAll "driver-last-location-update"
```

Comment:
Please use sorted set Redis data structure and commands `ZADD` and `ZRANGEBYSCORE`, to add driver ids into set sorted based on timestamp and lookup older driver ids with range on the same timestamp